pubspec.yaml

```diff
@@ -14,12 +14,17 @@ dependencies:
     path: packages/presentation
 
   cupertino_icons: ^1.0.2
+  get_it: ^7.2.0
+  injectable: ^1.5.3
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
 
   flutter_lints: ^2.0.0
+  build_runner: 2.2.0
+  injectable_generator: 1.5.4
```

Comment:
same, u don't generating any code in this module

Reply:
removed it